pyproject.toml

```diff
@@ -55,27 +56,28 @@ dev = [
 pinned = [
     "asgiref==3.6.0",
     "Django==4.2",
-    "psycopg==3.1.8",
-    "psycopg-binary==3.1.8",
+    "psycopg==3.1.9",
+    "psycopg-binary==3.1.9",
     "sqlparse==0.4.4",
     "typing_extensions==4.5.0",
 ]
 dev-pinned = [
     "alabaster==0.7.13",
-    "astroid==2.15.3",
+    "astroid==2.15.4",
     "Babel==2.12.1",
     "black==23.3.0",
     "build==0.10.0",
+    "bumpver==2022.1120",
     "certifi==2022.12.7",
     "charset-normalizer==3.1.0",
     "click==8.1.3",
     "colorama==0.4.6",
-    "coverage==7.2.3",
+    "coverage==7.2.5",
     "cssbeautifier==1.14.7",
     "dill==0.3.6",
-    "djlint==1.23.1",
-    "EditorConfig==0.12.3",
+    "djlint==1.25.0",
```

Comment:
When you update djlint, could you also update the corresponding pre-commit hook?